src/utils/store/getIllosInProject.js

```diff
@@ -0,0 +1,21 @@
+import store from '../../store';
+
+/**
+ * Get illustrations in a project
+ *
+ * @param {string} projectId - The unique identifier of the project
+ * @returns {Promise<Array|undefined>} A promise that resolves with an array of
+ * illustrations belonging to the specified project. If no projectId is
+ * provided, the promise resolves with 'undefined'.
+ */
+export default async function getIllosInProject(projectId) {
+  if (!projectId) {
+    return undefined;
+  }
+
+  const entities = await store.entities.get();
+  return entities
+    .filter(
+      ([, entity]) => entity.project === projectId,
```

Comment:
If we keep this, we should parse the entity ID instead of doing this, to keep things consistent.